src/python-common/requirements.txt

```diff
@@ -1,6 +1,4 @@
 six
```

Comment:
And... this one could also (at some time) be removed (extra work required).

Reply:
> It'll probably will be a FreeBSD/Ports/node/.... problem.
> But external installation of the dashboard (the user has to do that on FreeBSD)
> requires python2 to be installed for (from memory) installation of gyp.

are you sure? node-gyp supports python3. https://github.com/nodejs/node-gyp#on-unix

> But this is all stuff the end user has to do himself after installation of the ceph package